docker-compose.yml

```diff
@@ -150,6 +154,7 @@ services:
       - lando-api.db
 
   lando-api.db:
+    platform: linux/x86_64
```

Comment:
What does this do?

Reply:
https://docs.docker.com/compose/compose-file/compose-file-v2/#platform

Reply:
> What does this do?

Many of the images were failing to be built or pulled on aarch64/arm64 (including some of the nginx/mysql/tinyproxy ones). The error is `failed to create LLB definition: no match for platform in manifest`. This fixes it by explicitly telling docker to pull the x86_64 images, as I understand. The long term fix is probably for all these images to have builds for arm64 in their manifest, though this seems to work without any issue.